core/src/main/scala/cilib/Diversity.scala

```diff
@@ -0,0 +1,26 @@
+package cilib
+
+import scalaz._, Scalaz._
+import spire.math.Numeric
+
+object Diversity {
+
+  def averageDiversity[F[_], A](x: F[Position[A]])(
+      implicit F: Foldable[F],
+      N: Numeric[A]): Double = { // Kinda a matrix of entiies
```

Comment:
Typo entiies

Reply:
Thanks - This shouldn't be in the PR. It's some local work that I'm not ready to add yet. Will remove.